tests/system/src/test/java/test/com/sun/glass/ui/mac/MacPasteboardTest.java

```diff
@@ -67,7 +67,7 @@ public static void teardown() {
 
     @Test
     public void testValidLocalImageURLMacPasteboard() throws Exception {
-        assumeTrue(PlatformUtil.isMac());
+        assumeTrue(PlatformUtil.isMac() && !PlatformUtil.isHeadless());
```

Comment:
`setup` and `teardown` should probably be similarly qualified with "not headless".